src/imperative/imperative.cc

```diff
@@ -353,6 +356,11 @@ std::vector<NDArray*> Imperative::Backward(
         << "There are no inputs in computation graph that require gradients.";
   }
 
+  if (backward_graph_dump_enabled_) {
+    std::cout << "Forward graph: " << std::endl;
```

Comment:
Can we leave this to the developer? I think we can just have a function DebugGraph() and not putting all the std::cout logic in a function block.

Reply:
ok